site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: advice
+title: WebAuthn explained
+description: An in-depth look at how WebAuthn can help you add biometric authentication to your website.
+author: Dan Moore
+image: advice/vendor-lockin/expert-advice-avoiding-authentication-system-lock-in-header-image.png
```

Comment:
Do we have another image for the header?

Reply:
Yes, that's an open task and will be completed before this is pushed live.